tcms/testruns/templates/testruns/get.html

```diff
@@ -260,6 +260,27 @@ <h5><span class="test-executions-count"></span> Results</h5>
                                                 <p class="test-execution-notes">
                                                     <strong>{% trans 'Notes' %}:</strong>
                                                 </p>
+                                                <div class="test-execution-information">
+                                                    <div style="font-weight: bold">{% trans 'Test Execution Information' %}</div>
+                                                    <span style="padding-right: 1.5rem;">
+                                                        <span style="font-weight: bold">
+                                                            {% trans 'Run Date' %}:
+                                                        </span>
+                                                        <span class="run-date"></span>
+                                                    </span>
+                                                    <span style="padding-right: 1.5rem;">
+                                                        <span style="font-weight: bold">
+                                                            {% trans 'Build' %}:
+                                                        </span>
+                                                        <span class="build"></span>
+                                                    </span>
+                                                    <span style="padding-right: 1.5rem;">
+                                                        <span style="font-weight: bold">
+                                                            {% trans 'Text Version' %}:
```

Comment:
Use `Text version` b/c it already exists and will minimize the number of strings.